### PR TITLE
Upgrade moor

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

**Comment by joykuotw** [comment]
Which flutter version are you using?
Can you get the app run in web?

**Comment by mysticeti** [comment]
I am currently running Flutter 2.5.0. Yes forgot to mention, the web version in the current moor is experimental but I will address the web issue in a future commit, as the current compilation issue is due to the moor plugin.

**Comment by joykuotw** [comment]
I will not merge this one if the new upgrade make the web fail to compile.

**Comment by mysticeti** [comment]
Ok, I will address the web compilation in this commit.

**Review by joykuotw — APPROVED** [review]

File: lib/database/database_page.dart [hunk 1]
```diff
@@ -48,7 +48,9 @@ class _DatabasePageState extends State<DatabasePage>
           break;
         case 1:
           fabOnPressed = () {
-            addPersonMoorDB(_personDaoMoor);
+            if (!kIsWeb) {
+              addPersonMoorDB(_personDaoMoor);
+            }
```
Comment (joykuotw) [inline lib/database/database_page.dart:53]: Does it mean the moor still not fully web support?

Comment (mysticeti) [reply lib/database/database_page.dart:53]: The support is experimental. That is why I did not include it until the web support becomes stable.

**Pull request merged**